src/pages/[entity]/index.tsx

```diff
@@ -1,24 +1,21 @@
-import React, { useState } from "react";
-
+import { Badge } from "@material-ui/core";
 import Fab from "@material-ui/core/Fab";
+import { makeStyles } from "@material-ui/core/styles";
+import { DataGrid, GridColDef } from "@material-ui/data-grid";
 import AddIcon from "@material-ui/icons/Add";
 import RemoveIcon from "@material-ui/icons/Delete";
-
-import Layout from "~/components/Layout";
-import prest from "~/lib/prest";
-
-import { useRouter } from "next/router";
+import { PRestQuery, PRestTableShowItem } from "@postgresrest/node";
 import { GetServerSideProps } from "next";
-import { PRestTableShowItem } from "@postgresrest/node";
-import { DataGrid, GridColDef } from "@material-ui/data-grid";
-import { makeStyles } from "@material-ui/core/styles";
-import { PRestQuery } from "@postgresrest/node";
-import { Badge } from "@material-ui/core";
+import { useRouter } from "next/router";
+import React, { useState } from "react";
```

Comment:
React should always be the first import

Reply:
why should react be the first import?

my editor is set to orderna imports